network/bridge_endpointclient_linux.go

```diff
@@ -76,45 +77,45 @@ func (client *LinuxBridgeEndpointClient) AddEndpoints(epInfo *EndpointInfo) erro
 func (client *LinuxBridgeEndpointClient) AddEndpointRules(epInfo *EndpointInfo) error {
 	var err error
 
-	log.Printf("[net] Setting link %v master %v.", client.hostVethName, client.bridgeName)
+	log.Logger.Info("Setting link master", zap.String("linkName", client.hostVethName), zap.String("bridgeName", client.bridgeName), zap.String("component", "net"))
 	if err := client.netlink.SetLinkMaster(client.hostVethName, client.bridgeName); err != nil {
 		return err
 	}
 
 	for _, ipAddr := range epInfo.IPAddresses {
 		if ipAddr.IP.To4() != nil {
 			// Add ARP reply rule.
-			log.Printf("[net] Adding ARP reply rule for IP address %v", ipAddr.String())
+			log.Logger.Info("Adding ARP reply rule for IP address", zap.String("address", ipAddr.String()), zap.String("component", "net"))
 			if err = ebtables.SetArpReply(ipAddr.IP, client.getArpReplyAddress(client.containerMac), ebtables.Append); err != nil {
 				return err
 			}
 		}
 
 		// Add MAC address translation rule.
-		log.Printf("[net] Adding MAC DNAT rule for IP address %v", ipAddr.String())
+		log.Logger.Info("Adding MAC DNAT rule for IP address", zap.String("address", ipAddr.String()), zap.String("component", "net"))
 		if err := ebtables.SetDnatForIPAddress(client.hostPrimaryIfName, ipAddr.IP, client.containerMac, ebtables.Append); err != nil {
 			return err
 		}
 
 		if client.mode != opModeTunnel && ipAddr.IP.To4() != nil {
-			log.Printf("[net] Adding static arp for IP address %v and MAC %v in VM", ipAddr.String(), client.containerMac.String())
+			log.Logger.Info("Adding static arp for IP address and MAC in VM", zap.String("address", ipAddr.String()), zap.String("MAC", client.containerMac.String()), zap.String("component", "net"))
 			linkInfo := netlink.LinkInfo{
 				Name:       client.bridgeName,
 				IPAddr:     ipAddr.IP,
 				MacAddress: client.containerMac,
 			}
 
 			if err := client.netlink.SetOrRemoveLinkAddress(linkInfo, netlink.ADD, netlink.NUD_PERMANENT); err != nil {
-				log.Printf("Failed setting arp in vm: %v", err)
+				log.Logger.Info("Failed setting arp in vm with", zap.Any("error:", err), zap.String("component", "net"))
```

Comment:
There's a convenience function for errors: `zap.Error` that should be used instead of `zap.Any`. Repeat this for all other instances of `zap.Any("error:", err)`.
```suggestion
				log.Logger.Info("Failed setting arp in vm with", zap.Error(err), zap.String("component", "net"))
```